### ISSUE TITLE
Multiple identical weapons are displayed as one.

### ISSUE THREAD

**Issue by AdamEternal** [Orcpub/orcpub#340]
**Describe the bug**
When you have multiple of the same weapon (e.g. two handaxes), they are displayed as only one copy of it in character viewer and printed character sheet. This does not affect your ability to equip multiple of the same weapon. (E.g. I get two handaxes. It doesn't indicate in the character viewer or printed character sheet that I have multiple handaxes. I can, however, equip a handaxe in both hands at once.)

**To Reproduce**
Add multiple of the same weapon to your inventory. This can be either from character options or from the "Weapons, select any number" tool.

**Expected behavior**
I expect to see how many copies of a weapon I have.

**Screenshots**
https://i.imgur.com/etTSjYk.png
https://i.imgur.com/uCQwUgH.png

**Desktop:**
 - OS: Windows 10
 - Browser: Google Chrome
 - Version 80.0.3987.149 (Official Build) (64-bit)

**Comment by codeGlaze** [comment]
Tagging @sudonotpseudo

**Comment by sudonotpseudo** [comment]
Yep, thanks @codeGlaze .  I'll tackle this Friday along with Issue #395 .
